ThunderstoreCLI/Commands/InstallCommand.cs

```diff
@@ -92,25 +96,39 @@ private static async Task<int> InstallZip(Config config, HttpClient http, GameDe
 
         if (dependenciesToInstall.Length > 0)
         {
-            var downloadTasks = dependenciesToInstall.Select(mod => DownloadTemp(http, mod.LatestVersion!)).ToArray();
+            double totalSize = dependenciesToInstall.Select(d => (double) d.Versions![0].FileSize).Sum();
+            string[] suffixes = { "B", "KB", "MB", "GB", "TB" };
```

Comment:
This formatting stuff should probably be a separate helper function.

Reply:
Moved this and the same code in the Publish command